test/e2e/jest-playwright.config.js

```diff
@@ -13,18 +13,40 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+// assume a single browser is passed at a given time
+const browser = process.env.BROWSER || 'chromium';
+
 module.exports = {
-  server: {
+  serverOptions: {
     command: `npm run start -- --config-server-port 10002`,
     port: 10002,
     protocol: 'http', // if default or tcp, the test starts right await whereas the dev server is not available on http
     launchTimeout: 60000, // high value mainly for GitHub Workflows running on macOS (slow machines) and to build the bundle before start
     debug: true,
+    usedPortAction: 'ignore', // your test are executed, we assume that the server is already started
   },
-  launch: {
-    dumpio: true,
+  launchOptions: {
     headless: process.env.HEADLESS !== 'false',
+    // TODO make browser logs work
+    logger: {
+      // Make browser logs visible
+      isEnabled: name => name === 'browser',
+      // eslint-disable-next-line no-console
+      log: (name, severity, message) => console.log(`${name} ${message}`),
+    },
     slowMo: process.env.SLOWMO ? process.env.SLOWMO : 0,
+    // TODO review options, seems not working on webkit
+    // the following is for chromium
     args: ['--disable-infobars', '--no-sandbox', '--disable-setuid-sandbox'],
```

Comment:
There are already some properties on Playwright launch to do that: [**chromiumSandbox**](https://github.com/microsoft/playwright/blob/v1.7.0/docs/api.md#browsertypelaunchoptions)

I'm not sure it is useful to keep them.

Reply:
I have removed all args and this works for all browsers
This seems ok for webkit as well which was unhappy with unknown args.